utils/stripQuotes.js

```diff
@@ -0,0 +1,14 @@
+// This removes beginning and quotes from a string, like "example" becomes example
+// This is mainly used for query string examples in HTTP calls, because adding a quote mark there will break the URL
+
+export const stripQuotes = (inputString) => {
+  if (
+    inputString &&
```

Comment:
Yes, but because we're not using TypeScript, somebody could mistakenly try to pass an object, array, etc. to this function.

Reply:
Yes I was talking about the line 6 only. Anyway no problem! Let's go ahead!

Reply:
Ohhh, I see what you mean! i.e., `typeof(blah) === string` already supersedes the truthiness of`blah`, so that line 6 isn't needed at all. Good catch, and sorry I misunderstood!